packages/components/src/combobox-control/index.js

```diff
@@ -246,11 +247,8 @@ function ComboboxControl( {
 								className="components-combobox-control__input"
 								instanceId={ instanceId }
 								ref={ inputContainer }
-								value={ isExpanded ? inputValue : currentLabel }
-								aria-label={
-									currentLabel
-										? `${ currentLabel }, ${ label }`
-										: null
+								value={
```

Comment:
Because of the conditional rendering, I needed to make the value also conditionally rendered or else screen reader could select the same option by mistake.

Reply:
> Because of the conditional rendering, I needed to make the value also conditionally rendered or else screen reader could select the same option by mistake.

I'm not sure I understand your explanation here, could you please rephrase?

Reply:
@ciampo Because 2 different labels are used, I needed to make sure the `value=""` attribute was in sync with the suggestions list. Since the suggestions list is conditionally rendered, I had to do the same to the `value=""` or it could contain a stale value initially which could cause the user to select the same option.